blink/hostfs.c

```diff
@@ -128,6 +128,21 @@ int HostfsInit(const char *source, u64 flags, const void *data,
   return -1;
 }
 
+int HostfsReadmountentry(struct VfsDevice *device, char **spec, char **type, char **mntops) {
+  struct HostfsDevice *hostfsdevice = (struct HostfsDevice *)device->data;
+  *spec = strdup(hostfsdevice->source);
+  if (*spec == NULL) {
+    return enomem();
```

Comment:
You can just return -1 in cases like this, because strdup already set the errno.